.github/ISSUE_TEMPLATE.md

```diff
@@ -11,7 +11,7 @@ SUMMARY_GOES_HERE
 
 ### Additional Details & Logs
 
-- Kong version (`$ kong version`)
```

Comment:
I would keep the version as sometimes the issue can be at runtime and thus the user has long lost its stdout output.